libparsec/crates/tests_macros/src/lib.rs

```diff
@@ -63,33 +63,13 @@ pub fn parsec_test(attr: TokenStream, item: TokenStream) -> TokenStream {
         }
     }
 
-    let mut quote_attrs = quote! {};
-
-    for attr in parsec_test_item.attrs {
-        quote_attrs = quote! {
-            #quote_attrs
-            #attr
-        }
-    }
-
-    let quote_block = if sig.asyncness.is_some() {
-        sig.asyncness = None;
-        quote! {
-            ::libparsec_tests_fixtures::tokio::runtime::Builder::new_current_thread()
-                .enable_all()
-                .build()
-                .unwrap()
-                .block_on(async {
-                    #quote_block
-                })
-        }
-    } else {
-        quote_block
-    };
+    let tokio_decorator = sig.asyncness.map(|_| quote!(#[cfg_attr(not(target_arch = "wasm32"), ::libparsec_tests_fixtures::tokio::test(crate = "::libparsec_tests_fixtures::tokio"))])).into_iter();
```

Comment:
Can you format it ?
`(crate = ...)` really necessary ?

Reply:
Yes, because the `tokio::test` will think the crate is imported as a directed dependency and it may not be the case.

Here we provide a path where the proc-macro can access tokio api

Reply:
Can you format the line ?

Reply:
```rust
let tokio_decorator = sig
    .asyncness.map(|_| quote! {
        #[cfg_attr(not(target_arch = "wasm32"), ::libparsec_tests_fixtures::tokio::test(crate = "::libparsec_tests_fixtures::tokio"))]
        })
        .into_iter();
```

Reply:
done